README.md

```diff
@@ -91,7 +91,7 @@ release.
 
 The latest container image can be found at:
 * `registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.9.0` (arch: `amd64`, `arm`, `arm64`, `ppc64le` and `s390x`)
```

Comment:
This needs to be changed to v2.9.2 as well

Reply:
🤦🏼 Done.